content/modules/upgrading-rs.md

```diff
@@ -70,27 +70,31 @@ When you upgrade the module for a database, you can either:
 1. To upgrade the module for the database, run:
 
     ```src
-    rladmin upgrade module db_name <database_name> module_name <module_name> version <new_module_version_number> module_args <module arguments>
+    rladmin upgrade db <database_name> and module module_name <module_name> version <new_module_version_number> module_args "<module arguments>"
     ```
 
+    - If you want to remove the existing module arguments, enter `""` without arguments.
```

Comment:
Is this true? Not saying it isn't, just didn't know

Reply:
@K-Jo Please confirm.